pillar_examples/automatic/hana.sls

```diff
@@ -1,55 +1,45 @@
-{% set primary_node = 'hana01' %}
-{% set secondary_node = 'hana02' %}
-
 hana:
   {% if grains['qa_mode']|default(false) is sameas true %}
   install_packages: false
   {% endif %}
   nodes:
-    - host: {{ primary_node }}
+    - host: "{{ grains['name_prefix'] }}01"
       sid: 'prd'
       instance: 00
-      password: 'SET YOUR PASSWORD'
+      password: 'YourPassword1234'
       install:
-        software_path: '/root/sap_inst/'
+        software_path: '/root/sap_inst'
         root_user: 'root'
-        {% if grains['provider'] == 'libvirt' %}
```

Comment:
I think without this lines the execution in Libvirt will fail. Why have you removed them?

Reply:
false manipulation :+1: 
thanks